### ISSUE TITLE
Classifier development 201223

### ISSUE THREAD

**Issue by ltindale** [TalhoukLab/OV_Histotypes#25]
**Create training datasets**

- [x] All codesets, 72 genes
- All samples EXCEPT CS3 TNCO (confirmation) and CS3 DOVE (validation)
- For duplicate samples keep only last run sample
- (may test and compare other training sets later e.g. CS2+3, CS3 only)
- Create table of histotype distribution for:
- [x] Training set - overall and separated by CS
- [x] Confirmation set (TNCO)
- [x] Validation set (DOVE)

**Classifier**

In addition to analysis currently in the RSF add:
(from Cathy's thesis so she should have code stored)

- [ ] Classifiers: random forest version of Hellinger Distance Decision tree

- [x] Subsampling: smote upsampling for bottom minority classes only (Cathy used bottom 2 histotypes - check the histotype distribution of the training set to see if there are still 2 clear minority classes)

- [x] Comparison statistics: G-mean

**Training**

- [x] Test classifier on all_CS training set
